tests/phpunit/data/themedir1/block-theme/theme.json

```diff
@@ -69,6 +69,11 @@
 				"filter": {
 					"duotone": "var(--wp--preset--duotone--custom-duotone)"
 				}
+			},
+			"my/third-party-block": {
```

Comment:
This will be ignored by any other tests because the block `my/third-party-block` won't be registered.